components/list/list-item-mixin.js

```diff
@@ -55,9 +55,21 @@ export const ListItemMixin = superclass => class extends LocalizeCoreElement(Lis
 			breakpoints: { type: Array },
 			/**
 			 * Whether to render the list-item with reduced whitespace.
+			 * TODO: Remove in favor of padding-type="slim"
 			 * @type {boolean}
 			 */
 			slim: { type: Boolean },
+			/**
+			 * How much padding to render list items with
+			 * One of 'normal'|'slim'|'none', defaults to 'normal'
+			 * @type {string}
+			 */
+			paddingType: { type: String, attribute: 'padding-type' },
```

Comment:
Nit: we try to keep the public and private properties sorted alpha (separately), for both here and when they are assigned in the constructor.